frontend/src/features/mapSideBar/containers/MapSideBarContainer.tsx

```diff
@@ -298,12 +298,14 @@ const MapSideBarContainer: React.FunctionComponent<IMapSideBarContainerProps> =
    */
   const ConditionalDeleteButton = () => (
     <>
-      {keycloak.hasClaim(Claims.ADMIN_PROPERTIES) ||
-        (keycloak.hasClaim(Claims.PROPERTY_DELETE) &&
-          ((parcelDetail?.agencyId && keycloak.agencyIds.includes(parcelDetail.agencyId)) ||
-            (buildingDetail?.agencyId && keycloak.agencyIds.includes(buildingDetail.agencyId))) && (
-            <DeleteButton onClick={() => setShowDelete(true)} title="Delete Property" />
-          ))}
+      {true &&
```

Comment:
User has `admin-properties` or they have `property-delete` and the property belongs to their agency or sub-agencies.